tests/fv3_conf/rrfs_warm_run.IN

```diff
@@ -0,0 +1,32 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+
+OPNREQ_TEST=${OPNREQ_TEST:-false}
+SUFFIX=${RT_SUFFIX}
+cp -r @[INPUTDATA_ROOT]/FV3_input_data_conus13km/INPUT/* INPUT/
+
+for x in global_glacier.2x2.grb global_h2oprdlos.f77 global_maxice.2x2.grb \
+      global_o3prdlos.f77 global_snoclim.1.875.grb global_zorclim.1x1.grb ; do
+  #global_soilmgldas.t126.384.190.grb
+  cp -f @[INPUTDATA_ROOT]/FV3_fix/"$x" .
+done
+
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/global* .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/CFSR.SEAICE.1982.2012.monthly.clim.grb .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/RTGSST.1982.2012.monthly.clim.grb .
```

Comment:
Sam, these two data are also available in FV3_fix.

Reply:
I am testing this change now.

Reply:
Fixed.